index.html

```diff
@@ -10,97 +10,133 @@
 
 </head>
 <body>
-
+      
        <header class="header-bar">
 
         <nav class="nav-bar"><h3 class="Left-Logo">My Logo</h3>
-          <a><img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"></a></nav>
-       
+          <a><img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"></a>
+          <ul class="nav-list">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+           </ul>
```

Comment:
[OPTIONAL] Kindly wrap the list items in an `a` tag and link them to the relevant parts of the page 🤗

For example:
```
<li><a href="id-of-portfolio-section-here">Portfolio</a></li>
```